packages/fake-driver/lib/driver.js

```diff
@@ -152,6 +152,14 @@ export class FakeDriver extends BaseDriver {
     return await super.deleteSession(sessionId);
   }
 
+  async getAppiumSessionCapabilities(sessionId) {
+    return {
```

Comment:
should we returns this.caps?

Reply:
Returned caps and a few extras as an example for now. No strong opinion but just an example to return an arbitrary values as example response value. (not only this.caps)